src/kirby/components/checkbox/checkbox.component.ts

```diff
@@ -0,0 +1,43 @@
+import { Component, OnInit, Input, Output, EventEmitter } from '@angular/core';
+
+declare var require: any;
+const style: any = require('sass-extract-loader!./checkbox.component.scss');
+
+@Component({
+  selector: 'kirby-checkbox',
+  templateUrl: './checkbox.component.html',
+  styleUrls: ['./checkbox.component.scss'],
+})
+export class CheckboxComponent implements OnInit {
+  @Input() checked: boolean;
+  @Input() color: string;
+  @Input() shape: string;
+  @Output() checkedChange = new EventEmitter<boolean>();
+
+  classes: string[] = [];
+
+  get nativeColor(): string {
+    if (this.color) {
+      return this.getThemeColor('kirby-' + this.color);
+    } else {
+      return this.getThemeColor('kirby-primary');
+    }
+  }
+
+  onChecked(event): void {
+    this.checked = event.value;
+    this.checkedChange.emit(this.checked);
+  }
+
+  constructor() {}
+
+  ngOnInit() {
+    this.classes.push('checkbox');
```

Comment:
Pushing these classes should done in `ngOnChanges`, do facilitate changing classes, based on `@Input()`-decorated properties.